server/src/ZetaSqlWrapper.ts

```diff
@@ -212,7 +227,17 @@ export class ZetaSqlWrapper {
       }
       this.registerTable(table);
```

Comment:
Looks like now if `!schemaUpdated` we already register table and we don't need to do it second time.